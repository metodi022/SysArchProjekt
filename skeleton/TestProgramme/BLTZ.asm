lui $1, 0xffff
bltz $1, end
ori $1, 3
end:
ori $1, 4
