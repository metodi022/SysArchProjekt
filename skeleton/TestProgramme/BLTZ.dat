0x3c01ffff
0x04200001
0x34210003
0x34210004
