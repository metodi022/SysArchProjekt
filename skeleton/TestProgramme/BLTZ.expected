@1 ffff0004